sre-cloud.md

```diff
@@ -0,0 +1,71 @@
+# S.R.E / Cloud Challenge
+
+## About the Challenge... 
+
+We have two challenges, the first one is an intermediate test, and the second is a more advanced: The both challenges is about defining an architecture for Zé at AWS, considering several requirements that are necessary to consider.
+
+Feel free to ask any question - we really want you to succeed and then, join us to help evolve our platform.
+
+## The First Challenge - Intermediate
+
+This first one is build an simple infrastructure as code (AWS). 
```

Comment:
Você já falou que é o primeiro, então "This first one" fica redundante. Pode deixar somente "Build a simple infrastructure as code" (Não se usa an antes de palavras com consoante)